server/migrations/20180907145330-create-likes-dislikes.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('LikesDislikes', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER,
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+    },
+    articleId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+    },
+    status: {
+      type: Sequelize.ENUM('liked', 'disliked'),
```

Comment:
you can change the ENUM value to `1` and `0`, so when an article is liked, it takes 1 and when it is disliked, it takes `0`

Reply:
Thanks for that. I have implemented the feedback